src/views/UserSearch/UserSearch.js

```diff
@@ -335,6 +335,7 @@ class UserSearch extends React.Component {
                               <div className={css.searchGroupWrap}>
                                 <SearchField
                                   aria-label="user search"
+                                  autoComplete="off"
```

Comment:
Does `autocomplete` need to be [all lower case](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete) or is that irrelevant?

Reply:
HTML attributes are not case-sensitive, so that is irrelevant. But anyway I already changed it to lowercase to follow one common pattern. Thanks.

Reply:
It looks like I got error that React requires to write attributes in camelCase style, so I remove back `autoComplete`